doc/rule-descriptions.md

```diff
@@ -86,9 +86,9 @@
 
```

Comment:
@WilcoFiers This file is also generated - should I remove these changes from the PR also?

Reply:
No, we try to keep this in sync with the .json files in the repo. We actually need the _template.json file updated too. Would you mind running `npm run build` to update that file as well?

Reply:
Ah understood - we want the generated files updated and committed, but they are not edited directly, and so they stay current with the values used elsewhere in code. Updated the PR with the generated _template file.

Reply:
@WilcoFiers I believe everything is committed and good to go at this point.